.github/workflows/bazel.yml

```diff
@@ -291,9 +291,37 @@ jobs:
           run: |
             cd /workspaces/magma
             bazel/scripts/remote_cache_bazelrc_setup.sh "${{ env.CACHE_KEY }}" "${{ env.REMOTE_DOWNLOAD_OPTIMIZATION }}" "${{ secrets.BAZEL_REMOTE_PASSWORD }}"
-            bazel build lte/gateway/release:sctpd_deb_pkg lte/gateway/release:magma_deb_pkg \
+            bazel run //lte/gateway/release:release_build \
               --config=production \
               --profile=Bazel_build_package_profile
+            mkdir packages
+            mv /tmp/packages/*.deb packages
+      - name: Publish magma and sctpd
+        if: github.event_name == 'push' && github.repository_owner == 'magma'
+        env:
+          ARTIFACTORY_URL: "https://artifactory.magmacore.org/artifactory/debian-test/pool"
+          DEBIAN_META_INFO: ";deb.distribution=focal-ci;deb.component=main;deb.architecture=amd64"
+        run: |
+          cd packages
+          for package in `ls -a1 *.deb`
+          do
+            echo "Pushing package ${package} to JFROG artifiactory: ${ARTIFACTORY_URL}"
+            HTTP_RESPONSE=$(curl --silent --write-out "HTTPSTATUS:%{http_code}" -uci-bot:${{ secrets.JFROG_CIBOT_APIKEYS }} -XPUT "${ARTIFACTORY_URL}/focal-ci/${package}${DEBIAN_META_INFO}" -T $package)
```

Comment:
- It might be easier to use the jfrog cli. We already do so: https://github.com/magma/magma/blob/5a72ab813e340ccd43ca6766362d96fb229d3b90/.github/workflows/build_magma_dep.yml#L80-L88
- jfrog cli or not, it would be nice, if we could mulit-line those many command line arguments for better readability.

Reply:
will have a look

Reply:
seems to work nice - thank you for the suggestion